src/Interpreters/OpenTelemetryLog.cpp

```diff
@@ -0,0 +1,46 @@
+#include "OpenTelemetryLog.h"
+
+#include <DataTypes/DataTypeArray.h>
+#include <DataTypes/DataTypeDate.h>
+#include <DataTypes/DataTypeDateTime.h>
+#include <DataTypes/DataTypesNumber.h>
+#include <DataTypes/DataTypeString.h>
+#include <DataTypes/DataTypeUUID.h>
+
+namespace DB
+{
+
+Block OpenTelemetrySpanLogElement::createBlock()
+{
+    return {
+        {std::make_shared<DataTypeUUID>(), "trace_id"},
+        {std::make_shared<DataTypeUInt64>(), "span_id"},
+        {std::make_shared<DataTypeUInt64>(), "parent_span_id"},
+        {std::make_shared<DataTypeString>(), "operation_name"},
+        {std::make_shared<DataTypeDateTime>(), "start_time"},
```

Comment:
Timestamps should get at least microsecond precision, at second level granularity it's much harder to figure out what is going on in terms of order of events.

Reply:
Extended them to microsecond precision.